Source/Core/Core/IOS/ES/ES.cpp

```diff
@@ -968,6 +970,40 @@ IPCCommandResult ES::GetConsumption(const IOCtlVRequest& request)
   return GetDefaultReply(IPC_SUCCESS);
 }
 
+static bool CanDeleteTitle(u64 title_id)
+{
+  // IOS only allows deleting non-system titles (or a system title higher than 1-101).
```

Comment:
Aw, I thought this was standard notation for title IDs... do you want me to change this to 0x1-0x101?